tests/test_features.py

```diff
@@ -151,17 +151,18 @@ class Foo(betterproto.Message):
     foo.baz = "test"
 
     # Other oneof fields should now be unset
-    assert foo.bar == 0
+    assert not hasattr(foo, "bar")
+    assert object.__getattribute__(foo, "bar") == betterproto.PLACEHOLDER
     assert betterproto.which_one_of(foo, "group1")[0] == "baz"
 
-    foo.sub.val = 1
```

Comment:
Is this a behaviour change?

Reply:
This PR makes it impossible to use `foo.sub.val = 1` when `foo.sub` is unset the group.

Reply:
I think this is a bit of a tradeoff. With this change the users of `betterproto` can no longer use the `foo.sub.val = 1` syntax for fields that are unset in groups, but this also means that there is less risk of them accidentally changing which field is set in a group.